### DISCUSSION TITLE
Ability to have multiple Y axis in a chart

### DISCUSSION THREAD

**Discussion by agileramblings** [datalust/seq-tickets#1455] [Help]
I often find that I want 2 series with different scales on a chart so having the ability to designate a series onto a second Y-axis would allow for this kind of visualization.

**Comment by andymac4182** [comment]
This would be great! Similar to some charting systems that let you define a left and right scale.

**Comment by nblumhardt** [comment]
We're really keen on this idea, but need to schedule it as part of a larger piece of dashboarding work. We'll move it to a discussion until there's a suitable window of opportunity :-)
